librosa/core/audio.py

```diff
@@ -345,20 +370,20 @@ def stream(path, block_length, frame_length, hop_length,
     ...                                              hop_length=2048,
     ...                                              center=False)
 
-    '''
+    """
 
     if not (np.issubdtype(type(block_length), np.integer) and block_length > 0):
-        raise ParameterError('block_length={} must be a positive integer')
+        raise ParameterError("block_length={} must be a positive integer")
     if not (np.issubdtype(type(frame_length), np.integer) and frame_length > 0):
-        raise ParameterError('frame_length={} must be a positive integer')
+        raise ParameterError("frame_length={} must be a positive integer")
     if not (np.issubdtype(type(hop_length), np.integer) and hop_length > 0):
-        raise ParameterError('hop_length={} must be a positive integer')
+        raise ParameterError("hop_length={} must be a positive integer")
```

Comment:
whoa, these raise lines are wrong!  there's no `.format(hop_length)` (etc) to interpolate the {} to actual values.

This is independent of the issue for this PR, but good to fix anyway.

Reply:
good catch ...